src/controllers/editProfile.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
+  res.render('editProfile');
```

Comment:
could we add the pageTitle for hbs? e.g., second arg to render {pageTitle: 'Edit Profile'}